packages/core/src/controllers/SiweController.ts

```diff
@@ -0,0 +1,87 @@
+import { subscribeKey as subKey } from 'valtio/utils'
+import { proxy, subscribe as sub } from 'valtio/vanilla'
+import type {
+  CreateSIWEMessageArgs,
```

Comment:
Let's standardise naming for these types i.e. have `SIWE` at the beginning for all i.e. `SIWECreateMessageArgs` / `SIWEVerifyMessageArgs` etc.